Libs/vtkAddon/vtkStreamingVolumeFrame.h

```diff
@@ -0,0 +1,100 @@
+/*=auto=========================================================================
+
+  Portions (c) Copyright 2005 Brigham and Women's Hospital (BWH) All Rights Reserved.
+
+  See COPYRIGHT.txt
+  or http://www.slicer.org/copyright/copyright.txt for details.
+
+  Program:   3D Slicer
+  Module:    $RCSfile: vtkStreamingVolumeFrame.h,v $
+  Date:      $Date: 2006/03/19 17:12:29 $
+  Version:   $Revision: 1.13 $
+
+=========================================================================auto=*/
+
+#ifndef __vtkStreamingVolumeFrame_h
+#define __vtkStreamingVolumeFrame_h
+
+// VTK includes
+#include <vtkObject.h>
+#include <vtkUnsignedCharArray.h>
+
+// vtkAddon includes
+#include "vtkAddon.h"
+
+/// \brief VTK object containing a single compressed frame
+class VTK_ADDON_EXPORT vtkStreamingVolumeFrame : public vtkObject
+{
+public:
+
+  enum
+  {
+    IFrame,
+    PFrame,
+    BFrame,
+  };
+
+  static vtkStreamingVolumeFrame* New();
+  vtkTypeMacro(vtkStreamingVolumeFrame, vtkObject);
+  void PrintSelf(ostream& os, vtkIndent indent) VTK_OVERRIDE;
+
+  vtkSetMacro(FrameType, int);
```

Comment:
Add documentation

Reply:
Could you please move documentation from member variables to get/set functions?
The reason is that when API documentation is generated, protected/private members (and their documentation) may not show up in the generated pages.